extensions/amp-video/0.1/amp-video.js

```diff
@@ -188,7 +188,9 @@ class AmpVideo extends AMP.BaseElement {
 
     this.configure_();
 
-    this.video_ = element.ownerDocument.createElement('video');
+    const video = this.createVideoOrDefinitionElement_();
+
+    this.video_ = video;
```

Comment:
Just curious: why not just use `this.video_` everywhere?

Reply:
Minified bundle size :) By using a scoped reference we leverage the minified reference name (which should be one or two chars) vs `_this3.video`.

It shouldn't make too much of a difference with this method alone but went ahead and made the change for consistency.